app/models/post.rb

```diff
@@ -1,2 +1,18 @@
-class Post < ActiveRecord
+class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+  belongs_to :author, class_name: 'User'
+
+  def increment_counter
```

Comment:
- [OPTIONAL] It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to database, then make the update_post_counter method private